src/userdb/userdbctl.c

```diff
@@ -1516,6 +1736,8 @@ static int run(int argc, char *argv[]) {
                 /* This one is a helper for sshd_config's AuthorizedKeysCommand= setting, it's not a
                  * user-facing verb and thus should not appear in man pages or --help texts. */
                 { "ssh-authorized-keys", 2,        VERB_ANY, 0,            ssh_authorized_keys },
+                /* Ditto, this is only for use systemd-userdb-load-credentials.service. */
```

Comment:
you added it to the help text anyway

Reply:
@poettering So is ssh-authorized-keys ;)

Reply:
yeah, you found a bug.

please update the comment, i guess the docs are here to stay

Reply:
Not addressed?